test_data/asm/block_to_block_with_bus_different_sizes.asm

```diff
@@ -32,9 +34,9 @@ machine Arith with
 
 machine Main with
     // HACK: We need to provide a range here, because otherwise the linker will set all machines to the same degree.
-    // Witgen will choose the degree 8.
-    min_degree: 4,
-    max_degree: 8,
+    // Witgen will choose the degree 4.
```

Comment:
Because each call happens twice (see above), I made the main machine smaller, so the submachine is still big enough.